charts/hocs-notify/values.yaml

```
@@ -33,6 +33,6 @@ hocs-generic-service:
         -Djavax.net.ssl.trustStore=/etc/keystore/truststore.jks
         -Dhttps.proxyHost=hocs-outbound-proxy.{{ include "cs-namespace" . }}.svc.cluster.local
         -Dhttps.proxyPort=31290
-        -Dhttp.nonProxyHosts=*.svc.cluster.local
+        -Dhttp.nonProxyHosts=AWS_SQS_NOTIFY_URL
```

This will still need to include the cluster hosts too, these docs suggest patterns should be separated by `|` https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html
```suggestion
        -Dhttp.nonProxyHosts=*.svc.cluster.local|$AWS_SQS_NOTIFY_URL
```